app/src/main/java/pl/ludek/smat/home/pool_office_client/domain/model/InitializationStateRelay.kt

```diff
@@ -0,0 +1,12 @@
+package pl.ludek.smat.home.pool_office_client.domain.model
+
+data class InitializationStateRelay(
+    var relayFirst:Boolean,
+    val relaySecond:Boolean,
+    val relayThird:Boolean,
+    val relayFourth:Boolean,
+    val relayFifth:Boolean,
+    val relaySixth:Boolean,
+    val relaySeventh:Boolean,
+    val relayEighth:Boolean,
```

Comment:
Please use an array instead of multiple booleans

Reply:
Seems to have been fixed.